pkg/packagekit/updates.less

```diff
@@ -23,6 +25,12 @@
     }
 }
 
+@media screen and (min-width: 640px) {
+    .pk-updates--header--auto {
+        justify-content: start;
```

Comment:
If the page is sufficiently wide, the toggle button snaps to the header, so it doesn't get lost by itself.

(Otherwise, in narrow pages, it uses the default for the `pk-updates--header` widget and aligns to the right.)